whipper/common/program.py

```diff
@@ -614,3 +617,12 @@ def writeLog(self, discName, logger):
         self.logPath = logPath
 
         return logPath
+
+    def writeToc(self, discName):
```

Comment:
I stumbled for a moment on the name of this-- what's actually happening is we're moving a previous created tocfile to the output directory, so maybe we should make the method name (and docstring if necessary) reflect this.

Reply:
I thought about this too: decided to use this name/style to mimic the existing code style. It's true in this case it's a bit deceptive: I'll amend this (+ add a docstring). What would be a good name for the method?

Reply:
What about `def move_tocfile`?